R/parseHeader.R

```diff
@@ -19,7 +19,11 @@
 #' f <- system.file("extdata", "example_data10.adat",
 #'                  package = "SomaDataIO", mustWork = TRUE)
 #' header <- parseHeader(f)
-#' header
+#' names(header)
+#' header$Header.Meta
```

Comment:
Would you mind putting blank spaces between each of the calls?
I think it makes the final `pkgdown` look nicer so the reader can delineate the call from the return as they scroll.
Thanks!